chia/util/cpus.py

```diff
@@ -0,0 +1,13 @@
+import os, multiprocessing
+
+def get_available_cpus():
+    cpu_count = multiprocessing.cpu_count()
+    try:
+        cpu_count = os.sched_getaffinity(0)
+    except AttributeError:
+        pass
+
+    if os.name == "nt" and cpu_count > 61:
```

Comment:
Is "nt" generic across all modern Windows versions?

Reply:
@hoffmang9 This exact check is in any case being done when trying to compile the chia blockchain, at the point where the code tries to import this [external file and this line](https://github.com/WoLpH/portalocker/blob/638f67af428413ef6b2a45633b6e4f034b11f272/portalocker/portalocker.py#L9).
I ran into a problem installing the whole thing, as described in https://github.com/Chia-Network/chia-blockchain/pull/5158.

Reply:
No worries - GH now requires approval for first time contributors to start full CI - hence you got a comment from some serious pain that caused that comment from me in the first place regarding the Python windows issues. My quick review does look like "nt" is safe indeed.